Interni/Verbali Interni/2020_12_18/informazioni.tex

```diff
@@ -0,0 +1,25 @@
+\newcommand{\DataMeeting}{2020-12-18}
+
+\newcommand{\DataDoc}{2020-12-19} 
```

Comment:
La data del documento principale (VI_2020_12_14) non corrisponde con la data della stesura del documento, cambia il nome del file in VI_2020_12_19